### ISSUE TITLE
[Masthead/Footer] Accessibility - Hyperlink has no link text, label or image with a text alternative

### ISSUE THREAD

**Issue by SR1947** [carbon-design-system/carbon-for-ibm-dotcom#6785]
IBM Equal Access Accessibility Checker
Hyperlink has no link text, label or image with a text alternative

There are 37 links in footer which have been shown as violation. Listing some of them. I think all the links in footer have problem.
List of elements having problem:

```<a href="https://www.ibm.com/products?lnk=fps" class="bx--link bx--footer__link" part="link" id="link">
<a href="https://www.ibm.com/industries?lnk=fps" class="bx--link bx--footer__link" part="link" id="link">
<a href="https://ibm.com/cloud/ai?lnk=fps" class="bx--link bx--footer__link" part="link" id="link">
<a href="https://www.ibm.com/blockchain?lnk=fps" class="bx--link bx--footer__link" part="link" id="link">
<a href="https://www.ibm.com/business-operations?lnk=fps" class="bx--link bx--footer__link" part="link" id="link">```

**Comment by annawen1** [comment]
The issue seems to be the IBM Equal Access Accessibility Checker is not able to detect slotted items within web-components as the link text is slotted. 
<img width="1079" alt="Screen Shot 2021-08-03 at 9 09 24 AM" src="https://user-images.githubusercontent.com/54281166/128020982-bc08b269-7a96-4f32-966a-cf9fae307aa5.png">

The IBM Equal Access Accessibility Checker tool would need to be updated to support web-components
